src/System.Runtime.Extensions/tests/System/IO/PathTestsBase.cs

```diff
@@ -205,6 +205,174 @@ public partial class PathTestsBase : RemoteExecutorTestBase
             { @"C:\\foo2", @"C:\" },
         };
 
+        public static TheoryData<string, string, string> GetFullPath_Windows_NonFullyQualified => new TheoryData<string, string, string>
```

Comment:
It doesn't make sense to put the data in the base class unless it needs shared. Put it down with the tests that it is for.